terraform/providers/digital-ocean/images-tester/tester-environment.tf

```diff
@@ -22,11 +22,10 @@ resource "digitalocean_droplet" "droplet" {
   region = "${var.location}"
   size   = "512mb"
 
-
   connection {
-      user = "packer"
-      type = "ssh"
-      private_key = "${file("~/.ssh/id_rsa")}"
-      timeout = "2m"
+    user        = "packer"
+    type        = "ssh"
+    private_key = "${file("~/.ssh/id_rsa")}"
```

Comment:
The key file is taken from the HOME of the host or the container ?

Reply:
yeah, this is a sskey is copied from localhost default path to container. but is can be documented too. Thanks for this advice! 👌

Reply:
Open a issue #19 to review the real need of the local ssh key on entire project.